src/parser/jobs/war/modules/SurgingTempest.tsx

```diff
@@ -1,25 +1,76 @@
 import {Trans} from '@lingui/react'
 import {DataLink} from 'components/ui/DbLink'
-import {Analyser} from 'parser/core/Analyser'
+import {ActionKey} from 'data/ACTIONS'
+import {Event} from 'event'
+import {filter} from 'parser/core/filter'
 import {dependency} from 'parser/core/Injectable'
 import {Actors} from 'parser/core/modules/Actors'
 import Checklist, {Rule, Requirement} from 'parser/core/modules/Checklist'
+import {Gauge as CoreGauge, TimerGauge} from 'parser/core/modules/Gauge'
 import {Invulnerability} from 'parser/core/modules/Invulnerability'
 import {Statuses} from 'parser/core/modules/Statuses'
+import Suggestions, {SEVERITY, TieredSuggestion} from 'parser/core/modules/Suggestions'
 import React from 'react'
 
-export class SurgingTempest extends Analyser {
+const SURGING_TEMPEST_GENERATORS: ActionKey[] = [
+	'MYTHRIL_TEMPEST',
+	'STORMS_EYE',
+	'INNER_RELEASE',
+]
+const SURGING_TEMPEST_GENERATION_AMOUNT = 30
+// Surging Tempest Extension
+const SURGING_TEMPEST_EXTENSION_AMOUNT = 10
+
+const ONE_MINUTE = 60000
+
+const SURGING_TEMPEST_EARLY_REFRESH_GRACE = 7500
```

Comment:
Technically this buffer only applies to Eye, since it's 5000 for Tempest, basically irrelevant for 6.1 IR, and then 5000 for pre-6.1 IR, because you get 3 GCDs consuming IR stacks so need 6 GCDs, but then you can ignore 4 due to the 10s extension from IR itself. Not super sold on that tho, so `SURGE_BUFFER` works too even if it's really more specific than that.
```suggestion
const EYE_BUFFER = 7500
```